communication/state.h

```diff
@@ -80,7 +80,7 @@ typedef enum
 typedef enum
 {
 	DESCENT_TO_SMALL_ALTITUDE,							///< First auto landing behavior
-	DESCENT_TO_GND										///< Second auto landing behavior, comes after DESCENT_TO_SMAL_ALTITUDE
+	DESCENT_TO_GND,										///< Second auto landing behavior, comes after DESCENT_TO_SMALL_ALTITUDE
```

Comment:
@dtsbourg Watch out,
didn't you push everything without checking what was different here ??
This change for example shouldn't be merged...